backend/.env.example

```
@@ -1,187 +1,76 @@
 # -----------------------------------------------------------------------------
 # GEMINI TTS CONFIGURATION (Alternative TTS Provider)
 # -----------------------------------------------------------------------------
-# Alternative to ElevenLabs for text-to-speech
-GEMINI_TTS_API_KEY=your_gemini_tts_api_key_here
-GEMINI_TTS_MODEL=gemini-2.0-flash-thinking-exp-1219
-# Turkish voice options: Puck, Despina, Callisto
+GEMINI_TTS_API_KEY=
+GEMINI_TTS_MODEL=gemini-2.5-flash-preview-tts
 GEMINI_TTS_VOICE_ID=Despina
-GEMINI_TTS_ENDPOINT=https://generativelanguage.googleapis.com/v1beta/models
+GEMINI_TTS_ENDPOINT=https://generativelanguage.googleapis.com/v1beta/openai/models
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The GEMINI_TTS_ENDPOINT key should go before the GEMINI_TTS_MODEL key

(UnorderedKey)